.circleci/config.yml

```diff
@@ -598,9 +598,9 @@ workflows:
           requires:
             - install
       - docs-verdaccio:
-          filters:
-            branches:
-              only: main
+          # filters:
+            # branches:
+              # only: main
```

Comment:
Will revert before merging. Just wanted to get a test app build: https://reactspectrum.blob.core.windows.net/reactspectrum/4e00561eb9b564b12a3d9ef850a590c12f213bb9/verdaccio/rac-spectrum-tailwind/index.html